docs/pages/design_tokens.js

```diff
@@ -1,13 +1,49 @@
 // @flow strict
+
 import type { Node } from 'react';
-import { Table, Text, Box } from 'gestalt';
+import { Flex, Table, Text } from 'gestalt';
+// $FlowExpectedError[untyped-import]
+import tokens from 'gestalt-design-tokens/dist/js/tokens.js';
 import MainSection from '../components/MainSection.js';
 import PageHeader from '../components/PageHeader.js';
 import CardPage from '../components/CardPage.js';
+import { TokenExample } from '../components/TokenExample.js';
+
+type Token = {|
+  name: string,
+  value: string,
+  comment: string,
+  category: string,
+|};
 
 const cards: Array<Node> = [];
 const card = (c) => cards.push(c);
 
+const tokenCategories = [
+  { name: 'Spacing', category: 'spacing', id: 'space' },
+  { name: 'Background color', category: 'background-color', id: 'background' },
+  { name: 'Text color', category: 'text-color', id: 'text' },
+];
+
+const tableHeaders = (
+  <Table.Header>
+    <Table.Row>
+      <Table.HeaderCell>
```

Comment:
nit: you can dedupe this markup a bit by mapping an array of the header names, since the markup is otherwise identical for each